src/modals/ValidatorDecentralization/index.tsx

```diff
@@ -0,0 +1,154 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { ButtonHelp, PolkadotIcon } from '@polkadot-cloud/react';
+import { clipAddress } from '@polkadot-cloud/utils';
+import { useEffect, useState, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHelp } from 'contexts/Help';
+import { CardHeaderWrapper, CardWrapper } from 'library/Card/Wrappers';
+import { DistributionDoughnut } from 'library/Graphs/DistributionDoughnut';
+import { formatSize } from 'library/Graphs/Utils';
+import { GraphWrapper } from 'library/Graphs/Wrapper';
+import { useSize } from 'library/Hooks/useSize';
+import { Title } from 'library/Modal/Title';
+import { StatusLabel } from 'library/StatusLabel';
+import type { ValidatorDetail } from '@polkawatch/ddp-client';
+import { useOverlay } from '@polkadot-cloud/react/hooks';
+import { usePolkawatchApi } from '../../contexts/Polkawatch';
+import { PolkawatchButton } from '../../library/PolkawatchButton';
+import { usePlugins } from '../../contexts/Plugins';
+import { useTheme } from '../../contexts/Themes';
+
+export const ValidatorDecentralization = () => {
+  const { t } = useTranslation('modals');
+  const { pwApi, networkSupported } = usePolkawatchApi();
+  const { options } = useOverlay().modal.config;
+  const { address, identity } = options;
+  const { openHelp } = useHelp();
+  const { mode } = useTheme();
+
+  const ref = useRef<HTMLDivElement>(null);
+  const size = useSize(ref.current);
+  const { height, minHeight } = formatSize(size, 300);
+  const [pwData, setPwData] = useState({} as ValidatorDetail);
+  const [analyticsAvailable, setAnalyticsAvailable] = useState(true);
+  const { pluginEnabled } = usePlugins();
+
+  const enabled = pluginEnabled('polkawatch');
+
+  // In Small Screens we will display the most relevant chart.
+  // For now, we are not going to complicate the UI.
+  const isSmallScreen = window.innerWidth <= 650;
+  const chartWidth = '330px';
+
+  useEffect(() => {
+    if (networkSupported && enabled)
+      pwApi
+        .ddpIpfsValidatorDetail({
+          lastDays: 60,
+          validator: address,
+          validationType: 'public',
+        })
+        .then((response) => {
+          setAnalyticsAvailable(true);
+          setPwData(response.data);
+        })
+        .catch(() => setAnalyticsAvailable(false));
+    else setAnalyticsAvailable(false);
+    return () => {};
+  }, [pwApi, address]);
+
+  return (
+    <>
+      <Title title={t('validatorDecentralization')} />
+      <div className="header">
+        <PolkadotIcon
+          dark={mode === 'dark'}
+          nocopy
+          address={address}
+          size={33}
+        />
```

Comment:
As per my apology at Sub0 about the regular breaking changes, I recently updated this component to be named `Polkicon`, it disables copying by default, and the background color now refers to `--background-default`. So we can omit the `dark` and `nocopy` props.

The resulting JSX is as follows:

```suggestion
        <Polkicon
          address={address}
          size={33}
        />
```

Reply:
this has been applied, no worries for refactoring code.